### PR TITLE
update variables names in _in_container_script_init.sh

### PR DESCRIPTION
**Pull request by OliverWBurke**
local variable switched to lower case

closes one of the items on :#10576

---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).

### CODE REVIEW

File: scripts/in_container/_in_container_script_init.sh [hunk 1]
```diff
@@ -19,10 +19,10 @@
 set -euo pipefail
 
 # This should only be sourced from in_container directory!
-IN_CONTAINER_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+in_container_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
```
Comment (potiuk) [inline scripts/in_container/_in_container_script_init.sh:22]:
```suggestion
IN_CONTAINER_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
readonly IN_CONTAINER_DIR
```

File: scripts/in_container/_in_container_script_init.sh [hunk 2]
```diff
@@ -19,10 +19,10 @@
...
 
 # shellcheck source=scripts/in_container/_in_container_utils.sh
-. "${IN_CONTAINER_DIR}/_in_container_utils.sh"
+. "${in_container_dir}/_in_container_utils.sh"
```
Comment (potiuk) [inline scripts/in_container/_in_container_script_init.sh:25]:
```suggestion
. "${IN_CONTAINER_DIR}/_in_container_utils.sh"
```

**Review by potiuk — CHANGES_REQUESTED** [review]
Great stuff. Though I think the env VAR is really a constant and should be read only

**Comment by kaxil** [comment]
Can you please rebase your PR on latest Master since we have applied [Black](https://github.com/apache/airflow/commit/4e8f9cc8d02b29c325b8a5a76b4837671bdf5f68) and [PyUpgrade](https://github.com/apache/airflow/commit/8c42cf1b00c90f0d7f11b8a3a455381de8e003c5) on Master.

It will help if your squash your commits into single commit first so that there are less conflicts.

**Pull request closed without merging**